lib/iris/tests/unit/lazy_data/test_as_concrete_data.py

```diff
@@ -62,6 +75,35 @@ def test_lazy_mask_data(self):
         self.assertMaskedArrayEqual(result, mask_data)
         self.assertEqual(result.fill_value, fill_value)
 
+    def test_lazy_scalar_proxy(self):
+        a = np.array(5)
+        proxy = MyProxy(a)
+        lazy_array = as_lazy_data(proxy)
+        self.assertTrue(is_lazy_data(lazy_array))
+        result = as_concrete_data(lazy_array)
+        self.assertFalse(is_lazy_data(result))
+        self.assertEqual(result, a)
+
+    def test_lazy_scalar_proxy_masked(self):
+        a = np.ma.masked_array(5, True)
+        proxy = MyProxy(a)
+        lazy_array = as_lazy_data(proxy)
+        self.assertTrue(is_lazy_data(lazy_array))
+        result = as_concrete_data(lazy_array)
+        self.assertFalse(is_lazy_data(result))
+        self.assertMaskedArrayEqual(result, a)
+
+    def test_dask_scalar_proxy_pass_through(self):
+        # This test will fail when using a version of Dask with
```

Comment:
A little bit naughty (hello future person who is going back and reading this PR).

Reply:
Hello